doc/saml-to-saml-adapter.md

```diff
@@ -0,0 +1,45 @@
+# saml-to-saml Adapter
+
+The might be a mismatch between the SAML profile of an IDP (1:1) or a federation (1:many) 
+and an SP with a limited SAML implementation. the proxy can convert the SAMl profile 
+to make both sides compatible. 
+
+## Support for a NameID format Emailaddress
+
+If the SP requires a NameID format EMailAddress add this configuration entry the SAMLFrontend 
+file:
+
+module: satosa.frontends.saml2.SAMLFrontend
+name: Saml2IDP
+config:
+  idp_config:
+    ...
+    service:
+      idp:
+        ...
+        name_id_format: ['urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress']
```

Comment:
Is `name_id_format` always a list? I think `pysaml2` [allows both a list and a value](https://github.com/rohe/pysaml2/blob/master/src/saml2/client_base.py#L318).